src/Time/Actions/SelfStartActions.hpp

```diff
@@ -183,8 +183,9 @@ struct Initialize {
   };
 
  public:
-  using simple_tags = typename StoreInitialValues<tmpl::push_back<
-      detail::vars_to_save<System>, ::Tags::TimeStep>>::simple_tags;
+  using simple_tags = typename StoreInitialValues<
+      tmpl::push_back<detail::vars_to_save<System>, ::Tags::TimeStep,
+                      ::Tags::Next<::Tags::TimeStep>>>::simple_tags;
```

Comment:
Could you update the doxygen that now `Next<TimeStep>` is also saved and why it's saved?